persistence-jdbc/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/jdbc/LagomDBComponents.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.scaladsl.persistence.jdbc
+
+import com.lightbend.lagom.internal.persistence.LagomDBApiProvider
+import play.api.{ Configuration, Environment }
+import play.api.db.{ ConnectionPool, DBApi, DBApiProvider, DBComponents }
+import play.api.inject.ApplicationLifecycle
+
+/**
+ * DB components (for compile-time injection).
+ */
+trait LagomDBComponents extends DBComponents {
+  def environment: Environment
```

Comment:
This new component must implement `DBComponents` because of backward compatibility. otherwise MiMa will complain that the jdbc component is not extending it anymore.